src/how-to/create-a-github-action.html

```diff
@@ -0,0 +1,161 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="How to create a GitHub action using a workflow and a repository secret"
+    />
+    <link rel="stylesheet" href="/src/styles/common.css" />
+    <title>How to create a GitHub action</title>
+  </head>
+  <body class="body">
+    <header>
+      <h1>Create a GitHub action</h1>
+    </header>
+    <main class="main">
+      <article>
+        <section aria-labelledby="pull-requests">
+          <h2 id="pull-requests">Pull requests</h2>
+          <p>
+            <em>Task</em>: create a GitHub action which automatically assigns a
+            new pull request (PR) to its author and creates a list of potential
+            reviewers from a permitted list.
+          </p>
+          <p>
+            <em>Note</em>: the examples link to the People and Code repository
+            for this website and are not public.
```

Comment:
I wonder if we can create a small repo that can be forked or cloned for people to play with.